src/main/util.ts

```diff
@@ -2,6 +2,7 @@ import {inspect} from 'util'
 
 export function karhuInspect(obj: any) {
   if (typeof obj === 'string') return obj
+  if (obj && typeof obj.toKarhuString === 'function') return obj.toKarhuString()
```

Comment:
Can use optional chaining here
```suggestion
  if (typeof obj?.toKarhuString === 'function') return obj.toKarhuString()
```